CHANGELOG.rst

```diff
@@ -30,6 +30,33 @@ History
 
 * **Breaking Change**:  Change signature of hook ``update_internal_deposition_layer``. Now, instead of the deposition thickness, it is returned the phase index of phase being deposited and the thickness variation rate.
 * **Breaking Change**:  Change signature of hook ``calculate_relative_emulsion_viscosity`` and function ``get_relative_emulsion_viscosity``. Indices of continuous and dispersed fields were removed.
+* **Breaking Change**: Change in ``PhysicsDescription``: split emulsion model into relative viscosity, droplet size, and inversion point models and a flag to activate the emulsion models.
+
+* Examples on how to update previous .alfacase files with emulsion model:
+	- Relative viscosity model::
+		- Before::
+			emulsion_model: taylor1932
+
+		- After::
+			emulsion_model_enabled: True
+			emulsion_relative_viscosity_model: taylor1932
```

Comment:
This is not rendering correctly:

![image](https://user-images.githubusercontent.com/1085180/167398583-e2d71c2d-2c2b-4697-829f-89394a561955.png)

(You can go on the menu for the file here and click on "View file")

You need to remove the double `::` from the items, and add a separating line between the `::` in Before/After:

```suggestion
	- Relative viscosity model:
		- Before::
		
			emulsion_model: taylor1932

		- After::
		
			emulsion_model_enabled: True
			emulsion_relative_viscosity_model: taylor1932
```

Same for the ones below

Reply:
Thanks a lot